pandas/tests/indexes/test_base.py

```diff
@@ -1915,7 +1915,12 @@ def test_get_value(self, index):
         values = np.random.randn(100)
         value = index[67]
 
-        tm.assert_almost_equal(index.get_value(values, value), values[67])
+        with pytest.raises(AttributeError, match="has no attribute '_values'"):
+            # Index.get_value requires a Series, not an ndarray
```

Comment:
shouldn't this rather be a TypeError / AssertionError?

Reply:
only if we add the checks inside get_value.  My thought is that for internal methods with annotated types, we should be able to assume they are called correctly.

Reply:
only if we add the checks inside get_value.  My thought is that for internal methods with annotated types, we should be able to assume they are called correctly.